PackageBuilder/Build.cs

```diff
@@ -348,6 +367,15 @@ async Task LoadPackageInfo(Dictionary<string, VpmPackageInfo> listSourceVpmPacka
 
                     Serilog.Log.Information($"Found {PackageName(manifest)} {manifest.Version}, adding to listing.");
                     result.Add(manifest);
+
+                    // if there are unknown dependency packages, warn
+                    if (!manifest.Dependencies.Keys.All(knownPackages.Contains))
+                    {
+                        // TODO: should this be error and omit from package listing?
+                        Serilog.Log.Warning(
+                            $"We found some missing dependency packages in {packageId} version {version}: " +
+                            string.Concat(", ", manifest.Dependencies.Keys.Where(dep => !knownPackages.Contains(dep))));
```

Comment:
I am having an issue with the Warn log, i'm having the Linq displayed Raw

```
  08:13:02 [INF] Found nadena.dev.modular-avatar (Modular Avatar) 1.8.2, adding to listing.
  Warning: We found some missing dependency packages in nadena.dev.modular-avatar version 1.8.2: , System.Linq.Enumerable+WhereEnumerableIterator`1[System.String]
```
In addition i had an error thrown if the vpmDependencies entry doesn't exists.
I had that issue with the package [nadena.dev.ndmf](https://github.com/bdunderscore/ndmf) on the release 1.2.4

Here are the logs i have while using your branch on my CI on this repo https://github.com/Spokeek/vpm-repository

[logs_54.zip](https://github.com/vrchat-community/package-list-action/files/13332281/logs_54.zip)